pkg/api-server/customization/customization_suite_test.go

```diff
@@ -80,9 +82,9 @@ func createTestApiServer(store store.ResourceStore, config *config_api_server.Ap
 		},
 		&test_runtime.DummyEnvoyAdminClient{},
 		builtin.TokenIssuers{
-			DataplaneToken:   builtin.NewDataplaneTokenIssuer(resManager),
-			ZoneIngressToken: builtin.NewZoneIngressTokenIssuer(resManager),
-			ZoneToken:        builtin.NewZoneTokenIssuer(resManager),
+			DataplaneToken:   builtin.NewDataplaneTokenIssuer(ctx, resManager),
+			ZoneIngressToken: builtin.NewZoneIngressTokenIssuer(ctx, resManager),
+			ZoneToken:        builtin.NewZoneTokenIssuer(ctx, resManager),
```

Comment:
Why do we need to wire these in now? Seems like the methods of this class already take a ctx and that this ctx isn't used anywhere

Reply:
You're right, I'll remove these.